dash-docs/protocol-documentation.md

```diff
@@ -77,7 +77,7 @@ Whenever a masternode comes online or a client is syncing, they will send this m
 
 | Field Size | Field Name | Data type | Description |
 | ---------- | ----------- | --------- | ---------- |
-| 41 | vin | [CTxIn](#ctxin) | The unspent output which is holding 1000 DASH
+| 36 | outpoint | [COutPoint](#coutpoint) | The unspent output which is holding 1000 DASH
```

Comment:
Nit: Field name is `masternodeOutpoint` in most other messages

Reply:
Yep, the corresponding class is a child of `CMasternode` which is a child of `masternode_info_t` and this field is defined there. Having `masternodeOutpoint` instead of simply `outpoint` might look a bit excessive (especially later in code when this field is used) that's why it was `vin` and not `vinMasternode` historically. But I'm open to changing it now and unifying the name across all classes/messages :) Let's see what @codablock thinks though.

Reply:
Ah, yep. I see what you mean. :+1:

Reply:
Ok, I'll merge it as is now then. We can always fix it later if we change our minds :)